clang/include/clang/Driver/Options.td

```diff
@@ -2355,6 +2355,14 @@ def fsymbol_partition_EQ : Joined<["-"], "fsymbol-partition=">, Group<f_Group>,
   Visibility<[ClangOption, CC1Option]>,
   MarshallingInfoString<CodeGenOpts<"SymbolPartition">>;
 
+def fatomic_EQ : CommaJoined<["-"], "fatomic=">, Group<f_Group>,
+  Visibility<[ClangOption, CC1Option]>,
+  HelpText<"Specify atomic codegen options as a comma-separated list of "
```

Comment:
Nit. It reads like "options for atomic codegen". I assume that the options are for the codegen to use on atomic operations."Specify codegen options for atomic operations" may work a bit better for that.

Reply:
will do